tests/_grpc_utils.py

```diff
@@ -48,22 +50,23 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self._stdout_thread.join()
 
     def _get_grpc_server_exe(self):
-        if os.name != "nt":
-            pytest.skip("Only supported on Windows")
-        import winreg
+        if sys.platform == "win32":
+            import winreg
```

Comment:
mypy recognizes `sys.platform` but not necessarily `os.name`

Also this relied on mypy knowing that pytest.skip never returns, but apparently it does know that: 
https://github.com/pytest-dev/pytest/blob/8a410d0ba60bca7073bf18a1503c59a3b5445780/src/_pytest/outcomes.py#L120

Reply:
...which explains all the "unreachable code" warnings for return statements after a pytest.skip